test/virtual-machines.js

```diff
@@ -0,0 +1,31 @@
+import vm from 'node:vm';
+import * as chai from '../index.js';
+
+const {assert} = chai;
+const vmContext = {assert};
+vm.createContext(vmContext);
+
+function runCodeInVm(code) {
+  vm.runInContext(code, vmContext);
+}
+
+describe('node virtual machines', function () {
+  it('throws', function() {
+    const shouldNotThrow = [
+      `assert.throws(function() { throw ''; }, /^$/);`,
+      `assert.throws(function() { throw new Error('bleepbloop'); });`,
+      `assert.throws(function() { throw new Error(''); });`,
+      // TODO (43081j): enable this test once check-error supports
+      // cross-vm `Error` objects

```

Comment:
Could we link to a issue on `chaijs/check-error` so it's easy to follow this work. In case this doesn't get done in the near future we then have captured the relevant context. If you got a PR practically ready then no worries of course.

Reply:
this is the one you published today 🎉 

i just need to update the branch i think and remove this comment

Reply:
Oh I see haha 👯